src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/util/PropertyUtil.java

```diff
@@ -101,11 +103,17 @@ private static Long getLong(PropertyKeys property) {
     public static ConnectionSettingsProvider readFromPropFile(){
 
         if(fileProps == null) {
+            InputStream resourceStream = null;
+
             try {
                 fileProps = new Properties();
-                fileProps.load(PropertyUtil.class.getClassLoader().getResourceAsStream(MAIN_CONFIG_FILE_NAME));
+                resourceStream = PropertyUtil.class.getClassLoader().getResourceAsStream(MAIN_CONFIG_FILE_NAME);
+
+                fileProps.load(resourceStream);
             } catch (IOException ex) {
                 throw new RuntimeException("Error occured while loading config file: " + MAIN_CONFIG_FILE_NAME + ", " + ex);
+            } finally {
+                IOUtils.closeQuietly(resourceStream);
```

Comment:
if input is null here, doesn't it throw an error?

Reply:
it has a null check inside the method